Creating posts on OLS's annual report: it's broken into 3 parts (#202)
open-life-science/open-life-science.github.io: _posts/2020-10-01-annual-report-part-3.md

---
layout: post
title: OLS Project and Community report - Part 3
authors:
- bebatut
- malvikasharan
- yochannah
image: https://images.unsplash.com/photo-1499244571948-7ccddb3583f1?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2089&q=80
photos:
  name: Ross Findon, Unsplash id: @rossf
  license: CC-BY
  url: https://unsplash.com
---

# Year one of Open Life Science (OLS) - Part 3

*This is the last part of the 3 parts OLS project and community report: **Where we want to go***

> ## Shifting “volunteer culture” of Open Science

Reviewer: ```suggestion
## Shifting "volunteer culture" of Open Science
```

The value we create in OLS is impossible without the hard work of our participants (co-founders/lead trainers, mentors, expert guest speakers and project leads), who engage with OLS beyond their usual responsibilities. The first cohort was run entirely on their volunteer-work and for the current cohort we are actively seeking funding to support involvement of our participants in Open Science. 

We teach self-care and mental health awareness as part of OLS, but without the resources to sustain us, it’s very hard to actually practice what we preach. In 2021, we want to run two cohorts with a fairer reward system that empowers our volunteer members and ensures they are able to contribute to Open Science in a sustained manner. We would like for them to receive an honorarium or small support stipend. 

> We would also like to offer small-scale funding to our mentees from low-income backgrounds by offering them access to basic logistics that can improve their experience. For example, headsets, broadband, and other supplies will ensure equitable access to our calls and additional incubator funding for projects. Often a “sticker and biscuits” fund can be incredibly impactful, paying the small expenses needed to kick off a new organisation, whether this is an organisation’s legal registration, support to buy domain names, Zoom rooms, or swag as thank-yous for your community members.

Reviewer: ```suggestion

We would also like to offer small-scale funding to our mentees from low-income backgrounds by offering them access to basic logistics that can improve their experience. For example, headsets, broadband, and other supplies will ensure equitable access to our calls and additional incubator funding for projects. Often a "sticker and biscuits" fund can be incredibly impactful, paying the small expenses needed to kick off a new organisation, whether this is an organisation's legal registration, support to buy domain names, Zoom rooms, or swag as thank-yous for your community members.
```

This positive culture change is necessary to prevent unnecessary emotional burden and unfair treatment of contributing members in Open Science caused by unpaid volunteer work.

> ### OLS curriculum: *not just ‘Open’, but ‘community-minded’*

Reviewer: ```suggestion
### OLS curriculum: *not just 'Open', but 'community-minded'*
```

> From the beginning, OLS has been encouraging interested applicants to design their project ideas to not just be ‘Open’ but community-minded. Documentation, reproducibility, transparent reporting and public engagement aspects are strongly integrated into our curriculum and we accept projects with both technical and non-technical scopes. In the first round, many leads of technical projects also reported that their work benefited from the community-mindset we teach in OLS. We intend to strengthen our curriculum to add technical skill-up training (as we currently do for version control), but continue to run the current and next cohorts of the program to support community-oriented projects. We hope this will create momentum in the communities that can benefit immensely from Open Science practices.

Reviewer: ```suggestion
From the beginning, OLS has been encouraging interested applicants to design their project ideas to not just be 'Open' but community-minded. Documentation, reproducibility, transparent reporting and public engagement aspects are strongly integrated into our curriculum and we accept projects with both technical and non-technical scopes. In the first round, many leads of technical projects also reported that their work benefited from the community-mindset we teach in OLS. We intend to strengthen our curriculum to add technical skill-up training (as we currently do for version control), but continue to run the current and next cohorts of the program to support community-oriented projects. We hope this will create momentum in the communities that can benefit immensely from Open Science practices.
```

Feedback from the participants in OLS-1 and OLS-2 will help us enhance the quality of the training as we prepare for the future cohorts. Furthermore, with the support of the EOSC-Life Training grant, we will develop a new module on [FAIR principles](https://www.go-fair.org/fair-principles/). 

> Meanwhile, we have also improved the FAIR-ness of the way we work by moving our shared note-taking from google-docs to HackMD, which can be integrated with cohort-specific GitHub repositories. All the website content is designed in a way that information inconsistencies and backlogs can be avoided.

Reviewer: ```suggestion

Meanwhile, we have also improved the FAIR-ness of the way we work by moving our shared note-taking from Google Docs to HackMD, which can be integrated with cohort-specific GitHub repositories. We are also working on [Schema.org](https://schema.org/) annotations of the material. All the website content is designed in a way that information inconsistencies and backlogs can be avoided.
```

> ## Reaching communities who are not here ‘yet’!

Reviewer: ```suggestion
## Reaching communities who are not here 'yet'!
```

> With the conclusion of the first cohort, OLS has established itself as a globally inclusive initiative and a unique mentoring program for life science researchers. Participants of our program come from different backgrounds, identities, domain knowledge, nationality, career-stage and institutional affiliation. To ensure that our mentees are supported appropriately, mentors are assigned based on their preferred language, time zones and areas of interests. We conduct our work under a respectful and inclusive environment and enforce our Code of Conduct in all its activities. For OLS-1, we used the auto-generated transcripts of our video recordings, but for this round we are using otter.ai for live transcription. We share our training materials, cohort call videos and all OLS resources under CC-BY license and actively promote them to reuse, remix, adapt and share. 

Reviewer: ```suggestion
With the conclusion of the first cohort, OLS has established itself as a globally inclusive initiative and a unique mentoring program for life science researchers. Participants of our program come from different backgrounds, identities, domain knowledge, nationality, career-stage and institutional affiliation. To ensure that our mentees are supported appropriately, mentors are assigned based on their preferred language, time zones and areas of interests. We conduct our work under a respectful and inclusive environment and enforce our Code of Conduct in all its activities. For OLS-1, we used the auto-generated transcripts of our video recordings, but for this round we are using [otter.ai](https://otter.ai/) for live transcription. We share our training materials, cohort call videos and all OLS resources under CC-BY license and actively promote them to reuse, remix, adapt and share. 
```

We also understand that active measures are more effective than passive advocacy of diversity and inclusion. Therefore, during our break between OLS-1 and OLS-2 we continued to proactively deliver workshops, public resources and seminars in order to reach out to researchers who are traditionally underrepresented in open science.

> We hosted 2 webinars before launching the OLS-2 call for application and presented the project report of the pilot program at the following international computation and education-related conferences (virtual conferences due to COVID-19 pandemic):

Reviewer: ```suggestion

We hosted 2 webinars before launching the OLS-2 call for application and presented the project report of the pilot program at the following international computation and education-related conferences (virtual conferences due to COVID-19 pandemic):
```

> - [Bioinformatics Community Conference 2020](https://bcc2020.github.io/): short talks and [3 workshops](https://bcc2020.sched.com/event/c44n/building-communities-with-open-source-open-science) (3 hours each) across 3 time zones.

Reviewer: ```suggestion
- **[Bioinformatics Community Conference 2020](https://bcc2020.github.io/)**: short talks and [3 workshops](https://bcc2020.sched.com/event/c44n/building-communities-with-open-source-open-science) (3 hours each) across 3 time zones.
```

> - Open Science Seminar at Eastern Africa Network for Bioinformatics Training (EANBiT) 2020: Participated in a training panel (2 hours)

Reviewer: ```suggestion
- **Open Science Seminar at Eastern Africa Network for Bioinformatics Training (EANBiT) 2020**: Participated in a training panel (2 hours)
```

> - Collaboration Workshop 2020 by Software Sustainability Institute, UK: lightning talk and a workshop (1.5 hours)

Reviewer: ```suggestion
- **Collaboration Workshop 2020 by Software Sustainability Institute, UK**: lightning talk and a workshop (1.5 hours)
```

> - CarpentryCon @ Home 2020: 2 workshops on [Open Leaders Fundamentals](https://2020.carpentrycon.org/schedule/#session-45) and [Ally skills](https://2020.carpentrycon.org/schedule/#session-46) (1.5 hours each)

Reviewer: ```suggestion
- **CarpentryCon @ Home 2020**: 2 workshops on [Open Leaders Fundamentals](https://2020.carpentrycon.org/schedule/#session-45) and [Ally skills](https://2020.carpentrycon.org/schedule/#session-46) (1.5 hours each)
```

> - Research Data Alliance Early Career and Engagement IG 2020: Webinar (1 hour)

Reviewer: ```suggestion
- **Research Data Alliance Early Career and Engagement IG 2020**: Webinar (1 hour)
```

We have learned the hard way that non-profit projects like OLS come with financial and personal challenges. These challenges will keep us from working towards our goals until we have developed a stable funding model and business strategy to ensure the sustainability of this project.

In the short term, we have planned to offer an Ally Skills training session that will be open to public registration as a way to raise some ready cash for the organisation. Attendance is free to OLS members, with varying ticket prices for individuals who are able to pay, and fee waivers available if interested members otherwise could not attend. A separate post will follow about our Ally Skills workshop to raise funding for OLS-2.

In the long term, we are exploring funding resources to meet our financial needs (as discussed in this report) to ensure that we can run and expand this program to its full potential.

We want to end this report expressing our gratitude to everyone who has been involved in the program and this community in any way. 

See you all online (and hopefully, someday in person, after the COVID-19 pandemic).

*Malvika, Yo and Bérénice*

**END OF THE REPORT**

> ***Please see the [Part 1 of this report](./2020-10-01-annual-report-part-1) and [Part 2 of this report](./2020-10-01-annual-report-part-2)***

Reviewer: ```suggestion
***READ MORE**: If you missed them, you can also read the first two parts of the report: [**Where we started**]({% link _posts/2020-10-01-annual-report-part-1.md %}) and [**How far we have come**]({% link _posts/2020-10-01-annual-report-part-2.md %}).*
```

---

*This is the last part of the 3 parts OLS project and community report: **Where we want to go**. In the first two parts, we covered: [**Where we started**]({% link _posts/2020-10-01-annual-report-part-1.md %}) and [**How far we have come**]({% link _posts/2020-10-01-annual-report-part-2.md %}).*

## Shifting "volunteer culture" of Open Science

The value we create in OLS is impossible without the hard work of our participants (co-founders/lead trainers, mentors, expert guest speakers and project leads), who engage with OLS beyond their usual responsibilities. The first cohort was run entirely on their volunteer-work and for the current cohort we are actively seeking funding to support involvement of our participants in Open Science. 

We teach self-care and mental health awareness as part of OLS, but without the resources to sustain us, it’s very hard to actually practice what we preach. In 2021, we want to run two cohorts with a fairer reward system that empowers our volunteer members and ensures they are able to contribute to Open Science in a sustained manner. We would like for them to receive an honorarium or small support stipend. 

> We would also like to offer small-scale funding to our mentees from low-income backgrounds by offering them access to basic logistics that can improve their experience. For example, headsets, broadband, and other supplies will ensure equitable access to our calls and additional incubator funding for projects. Often a "sticker and biscuits" fund can be incredibly impactful, paying the small expenses needed to kick off a new organisation, whether this is an organisation's legal registration, support to buy domain names, Zoom rooms, or swag as thank-yous for your community members.

Reviewer 2: ```suggestion
We would also like to offer small-scale funding to our mentees: 
1. To those from low-income backgrounds by offering them access to basic logistics that can improve their experience. For example, headsets, broadband, and other supplies will ensure equitable access to our calls. 
2. Additional incubator funding for projects. Often a "sticker and biscuits" fund can be incredibly impactful, paying the small expenses needed to kick off a new organisation, whether this is an organisation's legal registration, support to buy domain names, Zoom rooms, or swag as thank-yous for your community members.
```

Feedback from the participants in OLS-1 and OLS-2 will help us enhance the quality of the training as we prepare for the future cohorts. Furthermore, with the support of the EOSC-Life Training grant, we will develop a new module on [FAIR principles](https://www.go-fair.org/fair-principles/). 

Meanwhile, we have also improved the FAIR-ness of the way we work by moving our shared note-taking from Google Docs to HackMD, which can be integrated with cohort-specific GitHub repositories. We are also working on [Schema.org](https://schema.org/) annotations of the material. All the website content is designed in a way that information inconsistencies and backlogs can be avoided.

## Reaching communities who are not here 'yet'!

> With the conclusion of the first cohort, OLS has established itself as a globally inclusive initiative and a unique mentoring program for life science researchers. Participants of our program come from different backgrounds, identities, domain knowledge, nationality, career-stage and institutional affiliation. To ensure that our mentees are supported appropriately, mentors are assigned based on their preferred language, time zones and areas of interests. We conduct our work under a respectful and inclusive environment and enforce our Code of Conduct in all its activities. For OLS-1, we used the auto-generated transcripts of our video recordings, but for this round we are using [otter.ai](https://otter.ai/) for live transcription. We share our training materials, cohort call videos and all OLS resources under CC-BY license and actively promote them to reuse, remix, adapt and share. 

Reviewer 2: ```suggestion
With the conclusion of the first cohort, OLS has established itself as a globally inclusive initiative and a unique mentoring program for life science researchers. Participants of our program come from different backgrounds, identities, domain knowledge, nationality, career-stage and institutional affiliation. To ensure that our mentees are supported appropriately, mentors are assigned based on their preferred language, time zones and areas of interests. We conduct our work under a respectful and inclusive environment and enforce our Code of Conduct in all its activities. For OLS-1, we used the auto-generated transcripts of our video recordings, but for this round we are using [otter.ai](https://otter.ai/) for live transcription. We share our training materials, cohort call videos and all OLS resources under the CC-BY license and actively promote them for reuse, remixing, adaptation and sharing. 
```

We also understand that active measures are more effective than passive advocacy of diversity and inclusion. Therefore, during our break between OLS-1 and OLS-2 we continued to proactively deliver workshops, public resources and seminars in order to reach out to researchers who are traditionally underrepresented in open science.

> We hosted 2 webinars before launching the OLS-2 call for application and presented the project report of the pilot program at the following international computation and education-related conferences (virtual conferences due to COVID-19 pandemic):

Reviewer 2: ```suggestion
We hosted 2 webinars before launching the OLS-2 call for applications and presented the project report of the pilot program at the following international computation and education-related conferences (virtual conferences due to COVID-19 pandemic):
```

- **Research Data Alliance Early Career and Engagement IG 2020**: Webinar (1 hour)

These engagements helped OLS become more visible across different research and open source communities. We will continue to do so to connect with those who are currently not in OLS, and Open Science at large.

## Planning for the long-term sustainability

Going forward, we would expand and improve our resources to develop customisable training and mentoring programs to meet the needs of individuals from different communities with varying levels of resources and institutional support available to them.
Currently, our time in OLS is not funded and is largely after-hours, and financially we are not in a position to onboard new members who can work with us to develop these curriculum. 

We have learned the hard way that non-profit projects like OLS come with financial and personal challenges. These challenges will keep us from working towards our goals until we have developed a stable funding model and business strategy to ensure the sustainability of this project.

> In the short term, we have planned to offer an Ally Skills training session that will be open to public registration as a way to raise some ready cash for the organisation. Attendance is free to OLS members, with varying ticket prices for individuals who are able to pay, and fee waivers available if interested members otherwise could not attend. A separate post will follow about our Ally Skills workshop to raise funding for OLS-2.

Reviewer 2: ```suggestion
In the short term, we are offering an Ally Skills training session that will be open to public registration as a way to raise some ready cash for the organisation. Attendance is free to OLS members, with varying ticket prices for individuals who are able to pay, and fee waivers available if interested members otherwise could not attend. A separate post will follow about our Ally Skills workshop to raise funding for OLS-2.
```
